src/Map.ts

```diff
@@ -0,0 +1,261 @@
+import { hasClass } from './decorators';
+import {is} from './object';
```

Comment:
Just refreshing the earlier line note here, that `{is}` should be `{ is }` here.